pkg/cmd/dedicated/deregister/deregister.go

```diff
@@ -115,15 +116,11 @@ func runDeRegisterClusterCmd(opts *options) error {
 func getListOfClusters(opts *options) ([]*clustersmgmtv1.Cluster, error) {
 	kfmClusterList, response, err := kafkautil.ListEnterpriseClusters(opts.f)
 	if err != nil {
-		if response != nil {
-			if response.StatusCode == 403 {
-				return nil, opts.f.Localizer.MustLocalizeError("")
-			}
-
-			return nil, fmt.Errorf("%v, %v", response.Status, err)
```

Comment:
Why was this removed, if you are not logged in and you run the command the CLI will panic because response will be nil